mmv1/templates/terraform/update_encoder/org_policy_custom_constraint.go.erb

```diff
@@ -0,0 +1,7 @@
+// need to send resource_types in all PATCH requests
```

Comment:
This is probably the biggest question mark for me. The `update_encoder` I have here does exactly what we need, but I'm curious if it is something we should be trying to avoid. Also, this is needed because the PATCH request expects all fields to be sent and re-validates them, which is an API pattern I think we would prefer to avoid.

Reply:
That's unusual that the PATCH expects a complete object. I'd expect it to have been a PUT API given that. No concerns on my end, thanks for the context.